server/pulp/server/webservices/views/search.py

```diff
@@ -0,0 +1,118 @@
+"""
+This module contains search View superclasses. Your code should subclass these to achieve maximum
```

Comment:
s/search View superclasses/the SearchView superclass/
s/code/view code/
s/these to achieve maximum searching capability/this to create a specific search view/